guidance/credential-sharing.md

```diff
@@ -1,27 +1,21 @@
 # Helpful Engineering Shared Credentials
 
-Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users using a tool called [LastPass](https://www.lastpass.com).
+Whenever possible, no account should be shared with other users.  However, in some instances (e.g. social media accounts, some email accounts, *et cetera*) it will be required to share a single set of credentials with multiple users by using a tool called **[1Password](https://1password.com)**.
 
- ⚠️ Credentials should **never** be shared outside LastPass: please **don't** send passwords in plaintext by any other means.
+The DevOps and Infosec teams maintain a master account where all credentials are stored and shared with the approved users. Passwords are being rotated regularly for security reasons, and the new ones will automatically synchronize in the other users' 1Password vaults. 
 
-## LastPass master account
-The DevOps and Infosec teams maintain a master account where all credentials are stored and shared with the approved users. Passwords are being rotated regularly for security reasons, and the new ones will automatically synchronize in the other users' LastPass vaults.  
+:warning: Credentials should **never** be shared outside the password manager: please **do not** send passwords in plaintext by any other means.
 
-## Creating a LastPass account
-If you do not have a lastpass account please [create one](https://lastpass.com/create-account.php).  **DO NOT FORGET YOUR LASTPASS PASSWORD!**  It cannot be reset by the infosec team or lastpass.  Your password is used to generate your encryption key, so it is unique to you.  We also recommend installing the lastpass browser extension to your favorite browser.
+## :book: Getting started
 
-After your account is created make sure to sign in, this will generate your encryption keys so passwords can be shared securely.  Send the email address you used to sign up to the infosec team ([#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z)) and they will share passwords with you.  If you are outside of the admin group in slack approval for access will be required.  The infosec team will reach out to the admin group for approval. 
+Just send your email address to the _Infosec_ team ([#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z)) and you'll be added to 1Password with access to the shared secrets you're allowed to use. If you already have a 1Password account, use its email address when requesting access.
```

Comment:
The old document stated that we would be collecting the users' email addresses through ([#skill-infosec](https://app.slack.com/client/TUTSYURT3/CV4TYGC1Z), but this seems like a bad security measure. Wouldn't be better to ask for direct messages?

Reply:
I don't think security through obscurity is a principle upon which we should rely.

Reply:
Which measure do you think is "security by obscurity"? I think that posting people's email addresses in a public channel is not a good practice, and, as we're hiding emails from the Slack profiles, we should also discourage publishing the email addresses on an open channel.

Reply:
We're not hiding the emails for security reasons, but for privacy reasons.  I think the instructions need to be more clear about the fact that it's a DM and how that could be done.

Reply:
> We're not hiding the emails for security reasons, but for privacy reasons.
👍 Good point. It definitely was a wrong word/concept choice<sup>1</sup>

>  I think the instructions need to be more clear about the fact that it's a DM and how that could be done.
I haven't changed the suggested method yet, so we could choose changing it to a direct message or even a Google/Airtable form. I don't know which method is going to offer a better user experience.

***
<sub>1. ...though in some (edge) cases, email addresses might also affect the security: today is so easy to lookup leaked passwords with an email address and perform credential stuffing attacks.</sub>